core/templates/dev/head/components/button-directives/hint-and-solution-buttons.directive.ts

```diff
@@ -89,7 +90,7 @@ angular.module('oppia').directive('hintAndSolutionButtons', [
             ctrl.activeHintIndex = index;
             var promise = (
               HintAndSolutionModalService.displayHintModal(index));
-            promise.result.then(null, function() {
+            promise.result.catch(function() {
```

Comment:
Can you explain this change (with links to references)?

Reply:
unneeded change, updated it ptal Thanks!